src/main/java/core/basesyntax/strategy/MyFileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyFileReaderImpl implements MyFileReader {
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
```

Comment:
You are correctly using try-with-resources here, which is good because it ensures that the BufferedReader is closed properly even if an exception occurs.